codegen/lib/graphql_swift_gen.rb

```diff
@@ -221,9 +228,113 @@ def deserialize_value_code(field_name, expr, type, untyped: true)
       raise NotImplementedError, "Unexpected #{type.kind} argument type"
     end
   end
-
+  
+  def generate_input_init(type) 
+    text = "public static func create("
+	input_fields = type.required_input_fields + type.optional_input_fields
+    input_fields.each do |field|
+      text << escape_reserved_word(field.camelize_name)
+      text << ": "
+      text << swift_input_type(field.type, wrapped: true)
```

Comment:
This looks like a painful way to generate code 😕  I think `graphql_swift_gen.rb` is more of a place to do some calculated logic rather then template the entire `init`. Having a function that is constantly appening to a string isn't nearly as readable or friendly as using embedded ruby. This should be in `type.swift.erb`.